src/current/cockroachcloud/architecture.md

```diff
@@ -78,18 +78,22 @@ All CockroachDB {{ site.data.products.cloud }} organizations are given 50 millio
 
 ### Autoscaling
 
-Serverless clusters also have the ability to scale to zero and consume no compute resources when there are no active queries. When there are no active queries, you will pay for the storage your app is using, but not for Request Units. To avoid wasted resources, CockroachDB {{ site.data.products.cloud }} automatically pauses Serverless clusters that are inactive, which is defined by having no connection to the cluster for five consecutive minutes. Once the user attempts to reconnect to the cluster, the cluster will automatically resume. Pausing, resuming, and scaling clusters is a fully-managed process and will not disrupt or affect the user experience. However, it is important for your application to have connection retry logic in the event of node restarts or network disruptions. For more information, see the [Production Checklist]({% link cockroachcloud/production-checklist.md %}).
+CockroachDB {{ site.data.products.basic }} clusters have the ability to scale to zero and consume no compute resources when there are no active queries. When there are no active queries, you will pay for the storage your app is using, but not for Request Units. To avoid wasted resources, CockroachDB {{ site.data.products.cloud }} automatically pauses {{ site.data.products.basic }} clusters that are inactive, which is defined by having no connection to the cluster for five consecutive minutes. Once the user attempts to reconnect to the cluster, the cluster will automatically resume. Pausing, resuming, and scaling clusters is a fully-managed process and will not disrupt or affect the user experience. However, it is important for your application to have connection retry logic in the event of node restarts or network disruptions. For more information, see the [Production Checklist]({% link cockroachcloud/production-checklist.md %}).
 
-The diagrams below shows how CockroachDB {{ site.data.products.serverless }} autoscales with your application's traffic:
+The diagrams below shows how CockroachDB {{ site.data.products.basic }} autoscales with your application's traffic:
 
 <img src="{{ 'images/cockroachcloud/serverless-low-traffic.png' | relative_url }}" alt="Serverless low traffic state" style="width:100%; max-width:800px" />
 
 <img src="{{ 'images/cockroachcloud/serverless-high-traffic.png' | relative_url }}" alt="Serverless scaling" style="width:100%; max-width:800px" />
 
+### Provisioned capacity
+
+Each CockroachDB {{ site.data.products.standard }} cluster is configured with a single capacity value (in Request Units/sec) indicating the maximum expected throughput the cluster will need over time. We reserve sufficient system infrastructure to support the workload based on the configured capacity value. This value can be scaled up/down as needed.
```

Comment:
Note that there is a limit to how many times per week the provisioned capacity can be *reduced* (it can be increased any number of times. The default limit is 3.

Reply:
Ah, right. Added.